steward/src/cork/client.rs

```diff
@@ -0,0 +1,27 @@
+use crate::{error::Error, prelude::APP};
+use abscissa_core::Application;
+use somm_proto::cork::{query_client::QueryClient, QueryCellarIDsRequest, QueryCellarIDsResponse};
+use tonic::{transport::Channel, Response, Status};
+
+/// Client wrapper used to refresh the cache
+pub struct CorkQueryClient {
```

Comment:
Should we also create a `CorkMsgClient` in here for the message sender for consistency?

Reply:
Sending uses the `deep_space::Contact`, I'm not sure adding a wrapper would provide any benefit?

Reply:
Not crucial, just would make the interface feel cleaner I think. It's something we can come back to later.